source/common/stats/statsd.cc

```diff
@@ -38,7 +38,8 @@ void Writer::write(const std::string& message) {
 
 UdpStatsdSink::UdpStatsdSink(ThreadLocal::SlotAllocator& tls,
                              Network::Address::InstanceConstSharedPtr address, const bool use_tag)
-    : tls_(tls.allocateSlot()), server_address_(std::move(address)), use_tag_(use_tag) {
+    : BaseSink(tls.allocateSlot()), tls_(tls.allocateSlot()), server_address_(std::move(address)),
```

Comment:
General comment: I wonder if we should make everyone pay for local histograms if they may not actually need them. Granted, they would now get them as part of admin output which is quite nice. I don't feel very strongly about this but throwing it out there for discussion.